hardware/i2c-shared.md

```diff
@@ -17,7 +17,7 @@ The V1.5 micro:bit has a footprint for two different motion sensors: one made by
 
 ### I2C block diagram
 
-![I2C block](/docs/hardware/assets/i2c-block.svg)
```

Comment:
Can you check if this image is used somewhere else?  if not we can remove it from the repo.

Reply:
grep found no references; removed with 65cb2e1.